docs/DeveloperGuide.md

```diff
@@ -35,4 +212,21 @@
 
```

Comment:
![image](https://github.com/nus-cs2113-AY2324S2/tp/assets/88656341/d6bc139b-72ec-4877-8eec-4301ce1d4c61)
Nun functional requirements is empty, can specify the constraints under which the system is developed and operated